src/lib/eina/eina_debug_bt.c

```diff
@@ -235,7 +237,9 @@ _signal_handler(int sig EINA_UNUSED,
    _bt_cpu[slot] = sched_getcpu();
 # ifdef HAVE_PTHREAD_GETCPUCLOCKID
    /* Try pthread_getcpuclockid() first */
-   pthread_getcpuclockid(self, &cid);
+   #ifndef _WIN32
+      pthread_getcpuclockid(self, &cid);
+   #endif
```

Comment:
what should happen on a win32 system?

Reply:
Something I couldn't do in win32

Reply:
Maybe you should add it to [issue 3](https://github.com/expertisesolutions/efl/issues/3), so we deal with it latter.